src/util/redux.js

```diff
@@ -0,0 +1,246 @@
+import {
```

Comment:
how does the team feel about this desctuctured import of lodash methods? I think it makes the code below less noisy, but i can change it if nobody else likes it